hdfs/client.py

```diff
@@ -497,6 +499,7 @@ def upload(self, hdfs_path, local_path, n_threads=1, temp_dir=None,
       completion, it will be called once with `-1` as second argument.
     :param cleanup: Delete any uploaded files if an error occurs during the
       upload.
+    :param timeout: upload timeouts.
```

Comment:
It would be good to mention the unit and match the form of the other options - can you update?